client/src/constants/api.js

```diff
@@ -15,5 +15,6 @@ export default {
   user: (id) => `${host}/AppUsers/${id}`,
   authorizationUrl: `${host}/Hooks/authorization-url`,
   setAuthorizationCode: `${host}/Hooks/set-authorization-code`,
-  playlists: `${host}/Hooks/playlists`
+  playlists: `${host}/Hooks/playlists`,
+  getPlaylist: `${host}/Hooks/get-playlist`
```

Comment:
I would prefer to have the playlist ID as a path variable, rather than query param, and also to use simply `playlist` instead of `get-playlist`. 

Can we make this:
`playlist: (id) => `${host}/Hooks/playlist/${id}``